webapp/lib/view.dart

```diff
@@ -139,7 +140,7 @@ class ActivePackagesViewPartial {
     _configureAction = new AnchorElement()
       ..classes.add('active-packages__package-action')
       ..text = 'Configure'
-      ..href = '#/configuration';
+      ..href = '#/batch-replies-configuration';
```

Comment:
We'll need to make this configurable, so that the "Urgent conversations (Active)" package points to `escalates-configuration`, and only "Batch replies (Week 12) (Active)" points to `batch-replies-configuration`. It might require substantial changes to the model, so maybe best done in a follow up PR?

Reply:
As we discussed I will do this in a new PR